### ISSUE TITLE
Plan for the remaining DL classes

### ISSUE THREAD

**Issue by aadilmehdis** [RoboticsIIITH/summer-sessions-2020#13]
Hi Folks,

Hope your'e all well. I hope you have all been enjoying the sessions on Deep Learning till now. We have a lot more in store for you. Before you know, you'll be coding up your own Neural Networks and be training them..

Now due to the constraint of time, we are forced to conduct two sessions on the 25th May 2020, i.e. today. We will cover one session in the morning from 12:00 p.m. to 2:00 p.m. and another session at 8:00 p.m. to 9:30 p.m. I really do hope you all will be comfortable with these timings. If there is any issue please mail me and we can work out something that works for all.

We will also have a hands on session on the 26th of May 2020. The timings for these will be discussed in todays class.

Best
AadilMehdi Sanchawala

**Comment by BonJovi1** [comment]
Hi folks, 

Tomorrow, we'll be having our final lecture on Concept 02: Deep Learning, from **5pm to 7pm**. We'll keep it pretty simple, straight-forward and sweet, as we may have bombarded a LOT at you today! 

From 5-5:30 pm, we'll take a look at **Regularisation**, **Learning Rate** Scheduling and **Transfer Learning** - some very important concepts in DL. 
And from 5:30 to 7pm, we'll have an amazing hands on session, where **you'll get to train your own neural network from scratch! :D** 

Apologies for 'overshooting' the time today (pun intended). We hope you were able to take away atleast 60-70% of whatever you learnt - the architecture of CNN's, the math behind back-propagation and the different optimisation algorithms we use to minimise the loss. 

See you tomorrow :)
